db/migrate/20150306162524_add_unlimited_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddUnlimitedToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :unlimited, :boolean
```

Comment:
To be safe use a default:

`add_column :users, :unlimited, :boolean, default: false`

Then everyone (including the existing clients won't be unlimited), and we can manually bump the ones we want.